src/database.c

```diff
@@ -453,6 +453,13 @@ hs_error_t print_database_string(char **s, u32 version, const platform_t plat,
 HS_PUBLIC_API
 hs_error_t hs_serialized_database_info(const char *bytes, size_t length,
                                        char **info) {
+    // Decode and check the header
```

Comment:
I removed check of `bytes` and `length`, because `db_decode_header` already checks it and other functions calling `db_decode_header` (e.g., `hs_deserialize_database`) do not check `bytes` and `length`.